src/rust/ide/enso-protocol/src/language_server.rs

```diff
@@ -56,55 +59,45 @@ impl Display for Path {
 
 /// Notification generated by the File Manager.
 #[derive(Clone,Debug,PartialEq)]
-#[derive(Serialize, Deserialize)]
+#[derive(Serialize,Deserialize)]
 #[serde(tag="method", content="params")]
 pub enum Notification {
     /// Filesystem event occurred for a watched path.
     #[serde(rename = "file/event")]
-    FilesystemEvent(FilesystemEvent),
-}
-
-
-
-// =======================
-// === FilesystemEvent ===
-// =======================
-
-/// Filesystem event notification, generated by an active file watch.
-#[derive(Clone,Debug,PartialEq)]
-#[derive(Serialize, Deserialize)]
-pub struct FilesystemEvent {
-    #[allow(missing_docs)]
-    // The serialization format requires the information to be wrapped around an "event" property.
-    pub event : FilesystemEventInfo
+    FileEvent {
+        /// The `file/event` notification input wrapper.
+        /// The serialization format requires the information to be wrapped into a field named "event".
+        /// This behavior is currently not specified by the specification and the issue has been raised
+        /// to address this: https://github.com/luna/enso/issues/707
+        /// TODO [mwu] Update as the issue is resolved on way or another.
```

Comment:
```suggestion
        // TODO [mwu] Update as the issue is resolved on way or another.
```
I think todos should be not a part of docs.